src/inc/jithelpers.h

```diff
@@ -403,6 +403,9 @@
 
 #endif // COR_JIT_EE_VERSION
 
+    JITHELPER1(CORINFO_HELP_JIT_PINVOKE_BEGIN, NULL, CORINFO_HELP_SIG_UNDEF, MDIL_HELP_UNDEF)
```

Comment:
Should we constrain these under a #ifdef  so that they get included when compiling Core RT?